lambda-cloudformation-custom-resource/src/main/scala/feral/lambda/cloudformation/package.scala

```diff
@@ -94,7 +93,7 @@ package cloudformation {
     }
   }
 
-  case class CloudFormationCustomResourceRequest[A](
+  final case class CloudFormationCustomResourceRequest[A](
       RequestType: CloudFormationRequestType,
```

Comment:
IMO these should be `camelCase`, but I didn't want to tackle that in this PR.

Reply:
Yeah this is @bpholt's baby, I promised him not to mess with it 😝

Reply:
Camel case is fine; IIRC it's only this way to make the decoder easier.

Reply:
gotchu, let's follow up in another PR